profiles/serializers.py

```diff
@@ -17,14 +15,13 @@ class Meta:
 
 
 class ProfileSerializer(serializers.ModelSerializer):
```

Comment:
Better rename it. I suggest `ProfileListSerializer` to avoid misapprehension in future.

Reply:
was renamed. Thanks